timml/stripareasink.py

```diff
@@ -250,8 +245,8 @@ def changetrace(
                 u = u1 * (1.0 + eps)  # Go just beyond circle
             else:
                 u = u2 * (1.0 + eps)  # Go just beyond circle
-            xn = x1 + u * (x2 - x1)
-            yn = y1 + u * (y2 - y1)
-            zn = xyzt1[2] + u * (xyzt2[2] - xyzt1[2])
+            x1 + u * (x2 - x1)
```

Comment:
These statements should probably be removed? Or commented?

Reply:
I'll remove them.